src/main/webapp/resources/js/apis/sequencing-runs/sequencing-runs.js

```diff
@@ -29,6 +29,11 @@ export const sequencingRunsApi = createApi({
       query: (runId) => ({
         url: `${runId}/sequenceFiles`,
       }),
+      transformResponse(response, meta) {
```

Comment:
Wondering if this should not be done through a utility class in the controller.  Reason I am asking is that another controller might want to get the sequencing files but not need this attribute.